sdk/communication/test-resources/test-resources.json

```diff
@@ -78,7 +78,7 @@
       },
       "COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING": {
           "type": "string",
-          "value": "[listKeys(resourceId('Microsoft.Communication/CommunicationServices',variables('uniqueSubDomainName')), '2020-08-20-preview').primaryConnectionString]"
+          "value": "[listKeys(resourceId('Microsoft.Communication/CommunicationServices',variables('uniqueSubDomainName')), '2021-10-01-preview').primaryConnectionString]"
```

Comment:
>2021-10-01-preview [](http://example.com/codeflow?start=125&length=18)

this should be 2023-03-31 also

Reply:
@usvoyager For connection string, I copied this version (2021-10-01-preview) referring to ones in other platforms (java, js and python). There are still few references in test-resources across all platforms that refers to version other than 2023-03-31. So should I update all of them to "2023-03-31" ?
For now, I just updated the reference where deprecated version was used (2020-08-20-preview to 2023-03-31)